staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -1535,6 +1535,7 @@ const (
 type VirtualMachineSpec struct {
 	// Running controls whether the associatied VirtualMachineInstance is created or not
 	// Mutually exclusive with RunStrategy
+	// Deprecated: use RunStrategy instead
 	Running *bool `json:"running,omitempty" optional:"true"`
```

Comment:
FWIW there are examples of folks moving the struct field name to `Deprecated${field}` already in our v1 API while keeping the json field name same:
```
$ git grep type\ Deprecated.* staging/src/kubevirt.io/api/
staging/src/kubevirt.io/api/core/v1/schema.go:type DeprecatedInterfaceSlirp struct{}
staging/src/kubevirt.io/api/core/v1/schema.go:type DeprecatedInterfaceMacvtap struct{}
staging/src/kubevirt.io/api/core/v1/schema.go:type DeprecatedInterfacePasst struct{}
```